### PR TITLE
Rewrite resume and view for experiment

### PR DESCRIPTION
**Pull request by ultmaster**

### CODE REVIEW

File: nni/experiment/config/base.py [hunk 1]
```diff
@@ -256,6 +256,8 @@ def _dict_factory(items):
     ret = {}
     for key, value in items:
         if value is not None:
+            # FIXME: none is not missing. Sometimes we need to dump none.
```
[the remark this answers is not in the document]
Comment (matluster) [reply nni/experiment/config/base.py:259]: This results in inconsistency when a config is loaded and dumped again.
But I'll add your intention in the comments.

[the remark this answers is not in the document]
Comment (liuzhe-lz) [reply nni/experiment/config/base.py:259]: In this design config files should never contain `null`, then it's consistent.

File: nni/experiment/config/utils/internal.py [hunk 2]
```diff
@@ -144,6 +144,7 @@ def guess_list_config_type(objs, type_hint, _hint_list_item=False) -> list[Confi
         try:
             configs = [cls(**obj) for obj in objs]
         except Exception:
+            # FIXME: The errors are eaten here. We should at least print one of them.
```
Comment (liuzhe-lz) [inline nni/experiment/config/utils/internal.py:147]: Well I think they are not "error", but "incorrect guess".

**Review by liuzhe-lz — APPROVED** [review]

File: nni/experiment/config/utils/internal.py [hunk 3]
```diff
@@ -144,6 +144,7 @@ def guess_list_config_type(objs, type_hint, _hint_list_item=False) -> list[Confi
         try:
             configs = [cls(**obj) for obj in objs]
         except Exception:
+            # FIXME: The reason why the guess failed is eaten here. We should at least print one of them.
```
Comment (QuanluZhang) [inline nni/experiment/config/utils/internal.py:147]: why not print?

**Review by QuanluZhang — APPROVED** [review]

**Pull request merged**